package.json

```diff
@@ -52,7 +52,7 @@
     "@react-native-firebase/crashlytics": "^12.3.0",
     "@react-native-firebase/perf": "^12.3.0",
     "@react-native-masked-view/masked-view": "^0.2.4",
-    "@react-native-picker/picker": "^1.9.11",
+    "@react-native-picker/picker": "^2.2.1",
```

Comment:
Should we update it to 2.3.1 now that this PR is merged? https://github.com/react-native-picker/picker/pull/376